### PR TITLE
Refactor/ner

### PR DESCRIPTION
**Pull request by mu-arkhipov**

### CODE REVIEW

File: deeppavlov/core/data/utils.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 from tqdm import tqdm
 import tarfile
 import gzip
+import zipfile
+import numpy as np
 import re
 import zipfile
```
Comment (yoptar) [inline deeppavlov/core/data/utils.py]: zipfile is imported twice

Comment (mu-arkhipov) [reply deeppavlov/core/data/utils.py]: fixed, thanks

File: deeppavlov/models/ner/network_refactored.py [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+from deeppavlov.core.layers.tf_layers import embedding_layer, character_embedding_network, variational_dropout
+from deeppavlov.core.layers.tf_layers import cudnn_bi_lstm, cudnn_bi_gru, bi_rnn, stacked_cnn
+from deeppavlov.core.models.tf_model import TFModel
+from deeppavlov.core.common.check_gpu import check_gpu_existance
+from deeppavlov.core.common.registry import register
+
+INITIALIZER = tf.orthogonal_initializer
+
+
+@register('ner_ref')  # TODO: rename
```
Comment (yoptar) [inline deeppavlov/models/ner/network_refactored.py]: This looks like a pretty important todo

File: deeppavlov/models/preprocessors/field_getter.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+@register('field_getter')
+class FieldGetter(Component):
+    def __init__(self, field, *args, **kwargs):
+        self.field = field
+
+    def __call__(self, batch: Union[List[Dict], Tuple[Dict]], **kwargs):
+        if isinstance(batch, (list, tuple)):
+            return [self(item) for item in batch]
+        else:
+            return batch[self.field]
```
Comment (yoptar) [inline deeppavlov/models/preprocessors/field_getter.py]: When is batch not a list or tuple?

File: deeppavlov/models/preprocessors/lazy_tokenizer.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+from deeppavlov.core.common.log import get_logger
+
+log = get_logger(__name__)
+
+
+@register('lazy_tokenizer')
+class LazyTokenizer(Component):
```
Comment (yoptar) [inline deeppavlov/models/preprocessors/lazy_tokenizer.py]: How is it lazy?

File: deeppavlov/models/preprocessors/mask.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+from deeppavlov.core.models.component import Component
```
Comment (yoptar) [inline deeppavlov/models/preprocessors/mask.py]: Add a licence, please

File: deeppavlov/models/preprocessors/lazy_tokenizer.py (hunk 4 above)
Comment (mu-arkhipov) [reply deeppavlov/models/preprocessors/lazy_tokenizer.py]: do not tokenize if it is already tokenized

File: deeppavlov/models/preprocessors/field_getter.py (hunk 3 above)
Comment (mu-arkhipov) [reply deeppavlov/models/preprocessors/field_getter.py]: what else it could be?

Comment (yoptar) [reply deeppavlov/models/preprocessors/field_getter.py]: Why is there this if then?

Comment (mu-arkhipov) [reply deeppavlov/models/preprocessors/field_getter.py]: my bad

**Pull request merged**